XenAdmin/TabPages/GeneralTabPage.cs

```diff
@@ -1830,21 +1830,19 @@ private string poolUpdateString(Predicate<Pool_update> predicate)
         /// </summary>
         /// <param name="host"></param>
         /// <returns></returns>
-        private List<KeyValuePair<String, String>> CheckHostPatchesRequiringReboot(Host host)
+        private List<KeyValuePair<String, String>> CheckHostUpdatesRequiringReboot(Host host)

```

Comment:
Could we also update this method's summary text to say updates instead of paches?